mwim.el

```diff
@@ -124,38 +124,63 @@ to the end of line."
           (goto-char eoc)))))
   (skip-chars-backward " \t"))
 
+(defun mwim-forward-line (arg)
+  "Move ARG - 1 lines forward (backward if ARG is negative).
+If ARG is nil or 1 do nothing."
+  (interactive "^p")
+  (setq arg (or arg 1))
+  (when (/= arg 1)
+    (let ((line-move-visual nil))
+      (forward-line (1- arg)))))
+
```

Comment:
This is because when you use `p` option of `interactive`, default value of `arg` is `1`. But I just realised that it's possible to use `P` in combination with `numberp` check. So yeah, since I also don't like using `2` to jump forward, I am getting rid of this function.